documents/motions/10.2019.2 - Disciplinary Committee.md

```diff
@@ -25,5 +25,7 @@ The WCA Disciplinary Committee is an Advisory Committee of the WCA.
       3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
       4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
 2. All WCA Staff must fully cooperate with investigations by the WCA Disciplinary Committee.
-3. All Registered Speedcubers should fully cooperate with investigations by the WCA Disciplinary Committee.
+3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary Committee.
+   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee in a timely manner.
```

Comment:
I feel "a timely manner" is quite vague. My personal opinion is that it would be good to have a definition of what this is (e.g. one week) that can be used. We don't want debates to just turn into an argument about what is/isn't a "timely manner". Although I'm not WDC so I'm happy to be corrected.

Reply:
Well, we can either debate what is a "timely manner" now or later. :P

Reply:
One week sounds reasonable to me :stuck_out_tongue:

Reply:
It's been a week! ;-)

Reply:
OK, let's go with one week.

Reply:
I agree with 1 week.

Reply:
I have already included this in a `Staff review` commit that I'll be pushing live later.